vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,46 @@
+#page-container .ag-theme-alpine.vizro {
+  --ag-background-color: var(--main-container-bg-color);
+  --ag-foreground-color: var(--text-primary);
+  --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(--text-primary);
+  --ag-header-background-color: var(--main-container-bg-color);
+  --ag-header-column-resize-handle-display: none;
+  --ag-font-size: 14px;
```

Comment:
Some of these properties here get overwritten by more specific CSS selectors e.g. the font-size. So you might have to specify them again below in of the classes.

Reply:
I think once he deleted #page-container there will be no need to specify them below. Is there a reason why we added  #page-container to the name `#page-container .ag-theme-alpine.vizro `?

Reply:
I think even if you remove the ID, you will still have to specify some of these attributes in the classes below e.g. remove the `#page-container` and double-check the font-size of the cell values. It's 15px instead of 14px because it gets overwritten by some of the more specific CSS selectors.

It's a bit fiddly but I think we have to try find the relevant CSS selector that's specific enough for us to change the property and still enables the user to overwrite it if they want to (e.g. by them using the important tag)